DQM/HcalMonitorTasks/src/HcalDigiMonitor.cc

```diff
@@ -570,6 +570,8 @@ void HcalDigiMonitor::analyze(edm::Event const&e, edm::EventSetup const&s)
 		  i<=FEDNumbering::MAXHCALuTCAFEDID; i++) {
 	  if (i>FEDNumbering::MAXHCALFEDID && i<FEDNumbering::MINHCALuTCAFEDID)
 		  continue;
+	  if (i<1118)
```

Comment:
a comment here may be helpful (e.g. ``` skip uTCA non-HF FEDs``` ).

this hardcoded choice is not good for maintenance: at some point the HBHE uTCA will need to be monitored as well.